src/eureka/S3_data_reduction/s3_reduce.py

```diff
@@ -340,6 +340,19 @@ def reduce(eventlabel, ecf_path=None, s2_meta=None, input_meta=None):
                 # Manually mask regions [colstart, colend, rowstart, rowend]
                 if hasattr(meta, 'manmask'):
                     data = util.manmask(data, meta, log)
+                
+                # Mask uncalibrated BG region for NIRSpec observations
+                # Code used for generating a calibrated stellar spectrum
+                if meta.convert_to_e is False and meta.inst == 'nirspec':
+                    cutoff = 1e-4
+                    log.writelog("  Masking manually identified bad pixels...",
+                                 mute=(not meta.verbose))
```

Comment:
This comment doesn't seem to belong here as these are not manually identified pixels being masked, right?